### PR TITLE
[Feature]實作解析參數與轉導至對應的Service

### PR DESCRIPTION
**Pull request by Naihua-TW**
# Checklist:
- [ ] My code follows the style guidelines of this project (IntelliJ Default)
- [ ] My changes generate no new warnings
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/main/java/tw/b2e/receipt/ReceiptRouter.java [hunk 1]
```diff
@@ -1,9 +1,65 @@
...
+
+import com.slack.api.bolt.request.builtin.SlashCommandRequest;
+
+import tw.b2e.common.Router;
+import tw.b2e.receipt.service.BaseService;
+import tw.b2e.receipt.service.HelpService;
+import tw.b2e.receipt.service.UnknownService;
+import tw.b2e.receipt.vo.CommandParam;
+import tw.b2e.receipt.common.CommandParamParser;
+
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/ReceiptRouter.java:24]: @Slf4j

可以考慮學一下Lombok，算是挺好的東西

File: src/main/java/tw/b2e/receipt/vo/CommandParam.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package tw.b2e.receipt.vo;
+
+import java.util.HashMap;
+import java.util.Map;
+
```
Comment (Jian-Min-Huang) [inline src/main/java/tw/b2e/receipt/vo/CommandParam.java:5]: same problem, how about Lombok 😏

**Review by Jian-Min-Huang — CHANGES_REQUESTED** [review]
主要是Lombok 😆

File: src/main/java/tw/b2e/receipt/ReceiptRouter.java (hunk 1 above)
Comment (Naihua-TW) [reply src/main/java/tw/b2e/receipt/ReceiptRouter.java:24]: 已改為lombok風格，再麻煩Review 謝謝

File: src/main/java/tw/b2e/receipt/vo/CommandParam.java (hunk 2 above)
Comment (Naihua-TW) [reply src/main/java/tw/b2e/receipt/vo/CommandParam.java:5]: 已改為lombok風格，再麻煩Review 謝謝

**Review by Jian-Min-Huang — APPROVED** [review]
LGTM

**Comment by Jian-Min-Huang** [comment]
因為已經衝突到無法解，直接關掉重做PR可能會好一點

**Pull request closed without merging**